lib/payload.json

```diff
@@ -137,6 +166,44 @@
             }
           },
           "additionalProperties": false
+        },
+        "water": {
+          "type": "object",
+          "properties": {
+            "leak": {
+              "description": "Water leaking (string)",
```

Comment:
Since the device that uses it outputs it as a string not a boolean:

https://github.com/TheThingsNetwork/lorawan-devices/blob/a15feaa9341c216d7862814415eda6a8a34bacd8/vendor/dragino/lwl03a-codec.yaml#L9-L13

Reply:
It's the job of the normalizer to normalize that.

Reply:
Oh right 😅